test/unit/math/p5.Vector.js

```diff
@@ -617,9 +617,9 @@ suite('p5.Vector', function() {
 
       test('should not change x, y, z if n is 0', function() {
         v.div(0);
-        expect(v.x).to.eql(1);
-        expect(v.y).to.eql(1);
-        expect(v.z).to.eql(1);
+        expect(v.x).to.eql(v.x);
+        expect(v.y).to.eql(v.y);
+        expect(v.z).to.eql(v.x);
```

Comment:
Wouldn't this always be true regardless?

Reply:
Not sure I understand your comment. The test started failing because we changed it so that instead of returning 1 if you divide by zero, we return the original vector.

Reply:
If the vector `v` is changed because of dividing by zero, `v.x == v.x`; if it isn't changed because of dividing by zero, still `v.x == v.x`. 

Unless the value of the vector `v` is cached beforehand with `const v2 = v.copy()` and comparing against that cached value, wouldn't any comparison of `v.x` with `v.x` always return true?

Reply:
Oops, duh! Looking at this more closely, that divide by zero test was failing because we weren't catching that case when there was a scalar sent to the function. I updated `div` and added some more tests for divide by zero as well.